docs/best-practices/crate-node.txt

```diff
@@ -1,434 +1,533 @@
+    Node is being repurposed as no-master and no-data. Clean-up of table data
+    is performed.
 
-However, if a node’s cluster has permanently failed then it may be desirable
-to try and move it into a new cluster. The ``crate-node detach-cluster`` command
-lets you detach a node from its cluster by resetting its cluster UUID.
-It can then join another cluster with a different UUID.
+    Do you want to proceed?
 
-For example, after unsafe cluster bootstrapping you will need to detach all
-the other surviving nodes from their old cluster so they can join the new,
-unsafely-bootstrapped cluster.
+    Confirm [y/N] y
+    Node successfully repurposed to no-master and no-data.
 
+.. _unsafe-bootstrapping-cluster:
 
-.. WARNING::
+Unsafely bootstrapping a cluster
+================================
 
-  Execution of this command can lead to arbitrary data loss. Only run this tool
-  if you understand and accept the possible consequences and have exhausted all
-  other possibilities for recovery of your cluster.
+Using the ``unsafe-bootstrap`` command, you can unsafely override the voting
+configuration of a cluster as if performing another cluster-bootstrapping
+process. This operation is useful if there is at least one remaining
+master-eligible node, but it is not possible to restart a majority of the nodes.
 
-The sequence of operations for using this tool are as follows:
+If you have a choice of nodes on which to run the ``unsafe-bootstrap`` command,
+choose the one with the highest term. The node's terms increase with
+each election term. Whereas the node with the highest term is the
+most current node, consisting of the most current data.
 
-* Make sure you have really lost access to every one of the master-eligible nodes
-  in the cluster, and they cannot be repaired or recovered by moving their data
-  paths to healthy hardware.
+If there is more than one node with the same term, choose the one with the
+highest version.
 
-* Start a new cluster and verify that it is healthy. This cluster may comprise
-  one or more brand-new master-eligible nodes, or may be an unsafely-bootstrapped
-  cluster formed as described above.
+.. NOTE::
 
-* Stop all remaining data nodes.
+  Choosing the node with the highest term or version identifies the node with
+  the freshest cluster state what minimizes the quantity of data that might be
+  lost.
 
-* On each data node, run the ``crate-node detach-cluster`` tool as shown
-  below. Verify that the tool reported Node was successfully detached from
-  the cluster.
+Before running the ``unsafe-bootstrap`` command
+-----------------------------------------------
 
-* If necessary, configure each data node to discover the new cluster.
+Adhere to the following procedure before running the ``detach-cluster`` command:
 
-* Start each data node and verify that it has joined the new cluster.
+The ``unsafe-bootstrap`` command makes sure the node used to bootstrap the
```

Comment:
I'm not sure what this means. but perhaps the intro section will make it clear once links/clarification is added